### PR TITLE
Update Dependencies and Fix Errors

### PR DESCRIPTION
**Pull request by WissamProgress**
Related to [https://github.com/telerik/kendo-react-private/issues/2052](https://github.com/telerik/kendo-react-private/issues/2052)

### CODE REVIEW

File: src/components/Dialog.tsx [hunk 1]
```diff
@@ -32,10 +33,10 @@ export const EditDialog = (props) => {
         style={{ width: '350px', margin: '25px', textAlign: 'center' }}
         className={'k-textbox'}
         onChange={handleInputChange}
-        />
+      />
       <DialogActionsBar>
-        <button value={'rename'} className='k-button k-state-selected' onClick={handleDialogClick}>Rename</button>
-        <button value={'cancel'} className='k-button' onClick={handleDialogClick}>Cancel</button>
+        <Button value={'rename'} className='k-button k-selected' onClick={handleDialogClick}>Rename</Button>
```
Comment (zdravkov) [inline src/components/Dialog.tsx]: Once we use our Button component we can rethink which classNames we use, too.

**Review by zdravkov — APPROVED** [review]
LG

File: src/components/Dialog.tsx (hunk 1 above)
Comment (WissamProgress) [reply src/components/Dialog.tsx]: Right, I will remove both `k-button` and `k-selected` classNames and use the [themeColor](https://www.telerik.com/kendo-react-ui/components/buttons/api/ButtonProps/#toc-themecolor) prop

Comment (WissamProgress) [reply src/components/Dialog.tsx]: fe579ca

**Review by zdravkov — APPROVED** [review]

**Review by vveesseelliinnaa — APPROVED** [review]

**Pull request merged**